backend/src/main/java/com/webappsbusters/packagemanagement/domain/Parcel.java

```diff
@@ -0,0 +1,153 @@
+package com.webappsbusters.packagemanagement.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity(name = "parcels")
+public class Parcel {
+
+    @Id
+    @GeneratedValue(generator = "uuid2")
+    @GenericGenerator(name = "uuid2", strategy = "uuid2")
+    @Column(name = "id",  unique = true, nullable = false, columnDefinition = "VARCHAR(36)")
+    private String parcelId;
+
+    @Column(name = "parcelStatus", columnDefinition = "varchar(20)")
+    private String status;
```

Comment:
I think we should discus with everyone if it should be enum, but for now it can stay as string